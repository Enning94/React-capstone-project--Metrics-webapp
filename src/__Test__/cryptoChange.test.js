import React from 'react';
import renderer from 'react-test-renderer';
import CryptoTrend from '../Components/cryptoChange';

test('CryptoTrend renders correctly for positive change', () => {
  const tree = renderer.create(<CryptoTrend change={10} />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('CryptoTrend renders correctly for negative change', () => {
  const tree = renderer.create(<CryptoTrend change={-10} />).toJSON();
  expect(tree).toMatchSnapshot();
});
