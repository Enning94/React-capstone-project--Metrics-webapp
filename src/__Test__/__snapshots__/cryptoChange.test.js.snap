// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CryptoTrend renders correctly for negative change 1`] = `
<svg
  className="trend-down"
  fill="none"
  height="1em"
  stroke="currentColor"
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth="2"
  style={
    Object {
      "color": undefined,
    }
  }
  viewBox="0 0 24 24"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="23 18 13.5 8.5 8.5 13.5 1 6"
  />
  <polyline
    points="17 18 23 18 23 12"
  />
</svg>
`;

exports[`CryptoTrend renders correctly for positive change 1`] = `
<svg
  className="trend-up"
  fill="none"
  height="1em"
  stroke="currentColor"
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth="2"
  style={
    Object {
      "color": undefined,
    }
  }
  viewBox="0 0 24 24"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="23 6 13.5 15.5 8.5 10.5 1 18"
  />
  <polyline
    points="17 6 23 6 23 12"
  />
</svg>
`;
